src/hyperion/experiment_plans/rotation_scan_plan.py

```diff
@@ -316,11 +320,28 @@ def rotation_with_cleanup_and_stage(params: RotationScan):
                 params.chi_start_deg,
                 group="move_gonio_to_start",
             )
-            yield from rotation_scan_plan(
-                composite,
-                params,
-                motion_values,
+            yield from bps.wait("move_gonio_to_start")
+
+            axis = composite.smargon.omega
+            # can move to start as fast as possible
+            # TODO get VMAX, see https://github.com/bluesky/ophyd/issues/1122
+            yield from bps.abs_set(
+                axis.velocity, motion_values.max_velocity_deg_s, wait=True
+            )
+            LOGGER.info(f"moving omega to beginning, {motion_values.start_scan_deg=}")
+            if params.take_snapshots:
+                yield from setup_oav_snapshot_plan(
+                    composite, params, motion_values.max_velocity_deg_s
+                )
+                yield from oav_snapshot_plan(composite, params, oav_params)
+
+            yield from bps.abs_set(
+                axis,
+                motion_values.start_motion_deg,
+                group="move_to_rotation_start",
+                wait=True,
             )
```

Comment:
Should: If the `setup_oav_snapshot_plan` is setting the velocity max itself then can we just put the velocity set and the move to start back in the `rotation_scan_plan`?